<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ke2182a</class>
 <widget class="QDialog" name="ke2182a">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 16pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>KE2182A</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>501</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Single Current Mode</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Multiple Current Mode</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Averaging points</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="NoofPoints">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>66</width>
       <height>23</height>
      </rect>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="value">
      <double>5.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>421</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Sampling points</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="NoofPoints_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>66</width>
       <height>23</height>
      </rect>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Averaging points</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="NoofPoints_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>66</width>
       <height>23</height>
      </rect>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
    </widget>
    <widget class="QComboBox" name="RangeType_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>mA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>μA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nA</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Range</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TO</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="SetAmplitude_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
    </widget>
    <widget class="QComboBox" name="RangeType_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>100</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>mA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>μA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nA</string>
      </property>
     </item>
    </widget>
    <widget class="QDoubleSpinBox" name="SetAmplitude">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Mode</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>261</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QDoubleSpinBox" name="NoofPoints_7">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>66</width>
       <height>23</height>
      </rect>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Averaging points</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>490</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
